quickstarts/dojo/config.yml

```diff
@@ -0,0 +1,46 @@
+slug: dojo
+
+description: |
+  ## Comprehensive monitoring quickstart for Dojo
+  Monitor your Dojo application performance in the most complex and distributed environments to deliver exceptional user experience by proactively identifying and resolving performance issues without any downtime.
+
+  ## Why monitor Dojo?
+  The performance monitoring tool for Dojo application helps you in monitoring the entire infrastructure and identifies the critical issues in your application. Dojo performance monitoring enables you to get a complete picture of Dojo errors, as well as know when and where your Dojo application slows down. 
+  Some key components are:
+  - Monitor  Dojo applications can be used to create rich and varied user interfaces running within a browser.
+  - Help you identify issues that are truly important and quickly resolve them. Gain knowledge of performance issues in advance and thus giving you ample time to avoid application downtime.
+  - Performance provides you with the information you need to not only prioritize critical performance issues, but also trace them down to the root cause and solve them more quickly.
+  - View the overall health of your production environment in order to quickly identify and resolve problems with crippled services.
+
+
+  ## What’s included in this quickstart?
+  New Relic's Dojo monitoring quickstart boasts instant full-stack observability out-of-the-box:
+
+  With this quickstart, you can:
+  - Easily monitor code-related insights that acquaint developers with the intricate details of their application’s health and status by providing detailed information on errors, database queries, and transaction traces.
+  - View interactive dashboards show your application's core web vitals,user time on the site and browser performance metrics like Initial page load, Route change, Throughput and Page views with Javascript errors.
```

Comment:
This needs to be update to be more Dojo-specific I think. As far as I can tell, it doesn't have much relationship to what I'm seeing in the screenshots.

"Use interactive, continually updating dashboards to see your app's performance, such as load time distribution and AJAX response times."